ci/check-signed-off-by.sh

```diff
@@ -12,17 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# this retrieves the merge commit created by GH
-parents=(`git log -n 1 --format=%p HEAD`)
+giturl=(`git config --get remote.origin.url`)
+repo='gitlab'
+if [[ "$giturl" == *"$repo"* ]]; then
+  parent=(`git log -n 1 --format=%p HEAD`)
```

Comment:
Does this result in a single element array on Gitlab?

Reply:
Yes, I was wondering what I could do about that. I've enabled Gitlab Runners on our fork and wanted to use mostly of the existent scripts like this one instead of duplicating them. I'm thinking about the alternatives.